desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -130,10 +147,35 @@ void TabDiveInformation::divesChanged(dive_trip *trip, const QVector<dive *> &di
 	case DiveField::WATER_TEMP:
 		ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 		break;
+	case DiveField::ATM_PRESS:
+		ui->atmPressVal->setText(ui->atmPressVal->text().sprintf("%d",current_dive->surface_pressure.mbar));
+		break;
 	case DiveField::DATETIME:
 		updateWhen();
 		break;
 	default:
 		break;
 	}
 }
+
+void TabDiveInformation::on_atmPressVal_editingFinished()
+{
+	int32_t atmpress;
+	if ((current_dive) && (!current_dive->userflags))
+	{
+		if (ui->atmPressType->currentIndex() == 1) {		// If altitude has been specified:
+			atmpress = (int)(0.5 + ui->atmPressVal->text().toFloat());	// get altitude from text box
+			if (prefs.units.length == units::FEET)		// if altitude in feet
+				atmpress = (int)(atmpress / 3.28084); 	// 	convert feet -> meters
```

Comment:
we have a `feet_to_mm()` helper. I'd prefer to use that then to hardcode another value here in the middle of the code